### ISSUE TITLE
Make documentation more accessible

### ISSUE THREAD

**Issue by mattgathu** [rust-lang/www.rust-lang.org#642]
# Content Feature Request
## Summary
Have a direct link to the documentation on the landing page.

## Motivation
I would imagine most users sometimes refer to the website to a quick reference to the documentation.
It is not currently easy to go to [https://doc.rust-lang.org](https://doc.rust-lang.org) from the current landing page.

## Drawbacks
None

## Rationale and alternatives
- Why is this design the best in the space of possible designs?

It provides _IMHO_ more value to Rust users than the current design. It essentially makes it
way easier to access documentation, something the old website design did as well.

- What other designs have been considered and what is the rationale for not
  choosing them?
NA
- What is the impact of not doing this?

More friction for users using the website, IMHO.

## Maintenance
How will this content be maintained? Who will do it?

Requires no maintenance.

## Unresolved Questions
What related issues do you consider out of scope for this feature that could be
addressed in the future independently of the solution that comes out of this
feature?
None

**Comment by mattgathu** [comment]
Just to shed more light, I'm attaching this collage to show comparison:
![pjimage](https://user-images.githubusercontent.com/1155192/49898305-fd986400-fe58-11e8-8757-dd03fc2df18f.jpg)

**Comment by ashleygwilliams** [comment]
the learn page has the documentation on it- we believe it's useful to have some scaffolding to help explain the extent of our documentation and their audiences. there *is* a link to documentation in the footer. just so i understand- is this asking to rename "Learn" to "Documentation"?

**Comment by Mark-Simulacrum** [comment]
We do link to what is essentially a version of the bookshelf directly from the front page through "Learn" on the top bar; as far as I can tell the content there is essentially equivalent to https://doc.rust-lang.org/. Do you think perhaps that link is sufficient? To me "documentation" / "reference" and "learn" are quite similar, though I can see how it doesn't jump out as much. I think it's possible that we could change "Learn" there to "Documentation" ?

**Comment by jacobpledger** [comment]
"Learn" and "Documentation"/"Reference" are very distinct in my mind. Learn is where I would look for a Getting Started tutorial - how to write hello world, etc. (That said, I like the current page). Once I had any level of experience with the language I would never click on "Learn" again, or at least that's not the heading I would be seeking.
To hijack and answer Ashley's question, I personally like the "Learn" page the way it is, but an additional direct "Reference" link to, say, https://doc.rust-lang.org/reference/index.html among the navbar would also be nice.

**Comment by pravdomil** [comment]
oh please make it DOCUMENTATION so that is 100% clear

**Comment by mattgathu** [comment]
> is this asking to rename "Learn" to "Documentation"?
**NO**. This was about reducing the number of hoops one goes through before getting to **The Standard Library docs**.  
In hindsight I do not know how to improve the website to make this better. I have personally moved away from using the website to access the stdlib docs. 

I'm closing this issue since it was biased to how I was personally using the website and I have since adjusted and it's no longer an issue. If you feel it's important for you to have this changed, kindly open another issue and maybe articulate it better than I did.
